main.go

```diff
@@ -65,5 +65,5 @@ func main() {
 		console.Fatalln("Go runtime version check failed.", err)
 	}
 
-	minio.Main(os.Args, os.Exit)
+	minio.Main(os.Args)
```

Comment:
I see `main.Main()` will be tested by `recover()` because `fatalIf()`, `fatalln()`  are used and `cli` package itself `panic()`.   Even if we pass a exit function, it won't help here.

In this patch, `os.Exit()` needs to be replaced with `panic()`.  I am doing that change.